docs/guide/blog.md

```diff
@@ -78,18 +107,35 @@ func NewHandler(k keeper.Keeper) sdk.Handler {
 }
 ```
 
-`case *types.MsgCreatePost` handles messages of type `MsgCreatePost`, calls `CreatePost` method and returns back the response.
+The `case *types.MsgCreatePost` statement handles messages of type `MsgCreatePost`, calls the `CreatePost` method, and returns back the response.
 
 Every module has a handler function like this to process messages and call keeper methods.
 
-### Processing Messages
+## Register the Query Handler 
```

Comment:
is this location the right place in the action steps to tell our users to register the query handler?